src/otx/algo/visual_prompting/zero_shot_segment_anything.py

```diff
@@ -745,7 +742,14 @@ def _customize_inputs(  # type: ignore[override]
         if self.training:
             # learn
             forward_inputs.update(
-                {"processed_prompts": self._gather_prompts_with_labels(inputs.prompts, inputs.labels)},
+                {
+                    "processed_prompts": self._gather_prompts_with_labels(
+                        inputs.labels,
+                        inputs.prompts,
+                        inputs.polygons,
+                        inputs.masks,
+                    ),
```

Comment:
It seems that `_gather_prompts_with_labels` function reuiqres all attributes of `ZeroShotVisualPromptingDataEntity`. If then, how about passing `inputs` rather than each attirbutes?

Reply:
Updated https://github.com/openvinotoolkit/training_extensions/commit/ab08d6845bdca69cecb90482049e6e32d07aa4fe